quickstart/java/android/keyword-recognizer/README.md

```diff
@@ -0,0 +1,40 @@
+# Quickstart: Keyword Recognition in Java on Android
+
+This sample demonstrates how to recognize keywords with Java using the Speech SDK for Android.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-java-android) on the SDK documentation page which describes how to build this sample from scratch in Android Studio.
+
+> **Note:**
+> this sample is *not* for the Speech Devices SDK and the Roobo device.
+> If you are looking for further information on these, visit the [Speech Devices SDK](https://docs.microsoft.com/azure/cognitive-services/speech-service/speech-devices-sdk) page.
+
+## Prerequisites
+
+* A PC (Windows, Linux, Mac) capable to run Android Studio.
+* Version 3.1 of [Android Studio](https://developer.android.com/studio/).
+* An ARM-based Android device (API 23: Android 6.0 Marshmallow or higher) [enabled for development](https://developer.android.com/studio/debug/dev-options) with a working microphone.
+
+## Build the sample
+
+> Note: more detailed step-by-step instructions are available [here](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-java-android).
+
+* **By building this sample you will download the Microsoft Cognitive Services Speech SDK. By downloading you acknowledge its license, see [Speech SDK license agreement](https://aka.ms/csspeech/license201809).**
+* [Download the sample code to your development PC.](/README.md#get-the-samples)
+* Open this folder as a project in Android Studio.
+* Press Ctrl+F9, or select **Build** \> **Make Project**.
+
+## Run the sample
+
+* Connect your Android device to your development PC.
+* Press Shift+F10, or select **Run** \> **Run 'app'**.
+* In the deployment target windows that comes up, pick your Android device.
+
+## Note on Android permission handling
+
+Please note that the Speech SDK requires two permissions, i.e., the INTERNET as well as the RECORD_AUDIO permission. As for Android API levels up to 22, requesting the permissions is handled at installation time but from API level 23 users are able to grant/revoke permissions at any time.
```

Comment:
>INTERNET [](start = 68, length = 8)

this is not in the manifest, also not needed for keyword recognizer, right?

Reply:
It is not. I kept that as it was talking about the SDK. Rephrased it to talk about the sample